### PR TITLE
Address TODO items in preparation for 3.0 beta (6/many)

### PR DESCRIPTION
**Pull request by koikonom**

### CODE REVIEW

**Review by mbfrahry — APPROVED** [review]
LGTM!

File: internal/services/apimanagement/api_management_resource.go [hunk 1]
```diff
@@ -320,14 +320,6 @@ func resourceApiManagementSchema() map[string]*pluginsdk.Schema {
 						Default:  false,
 					},
 
-					// TODO 4.0: change this from enable_* to *_enabled
-					"enable_frontend_ssl30": {
```
Comment (katbyte) [inline internal/services/apimanagement/api_management_resource.go:324]: was this supposed to be removed?

**Review by katbyte — APPROVED** [review]
aside from one comment LGTM 💯

File: internal/services/apimanagement/api_management_resource.go (hunk 1 above)
Comment (koikonom) [reply internal/services/apimanagement/api_management_resource.go]: Nope... I had to rebase my branch and this got removed by accident. Went over the diff with main and fixed this and a couple more issues.

**Pull request merged**